test/JDBC/expected/smalldatetime_date_cmp-vu-verify.out

```diff
@@ -0,0 +1,3863 @@
+~~END~~
+
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.224 ms
+~~END~~
+
+
+SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col = CAST('2023-06-17' AS SMALLDATETIME);
+GO
+~~START~~
+text
+Query Text: SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col = CAST('2023-06-17' AS SMALLDATETIME)
+Aggregate
+  ->  Seq Scan on date_smalldatetime_cmp_t1
+        Filter: (date_col = '2023-06-17 00:00:00'::smalldatetime(0) without time zone)
+~~END~~
+
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.175 ms
+~~END~~
+
+
+SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col > CAST('2023-06-16' AS SMALLDATETIME);
+GO
+~~START~~
+text
+Query Text: SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col > CAST('2023-06-16' AS SMALLDATETIME)
+Aggregate
+  ->  Seq Scan on date_smalldatetime_cmp_t1
+        Filter: (date_col > '2023-06-16 00:00:00'::smalldatetime(0) without time zone)
+~~END~~
+
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.269 ms
+~~END~~
+
+
+SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col >= CAST('2023-06-16' AS SMALLDATETIME);
+GO
+~~START~~
+text
+Query Text: SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col >= CAST('2023-06-16' AS SMALLDATETIME)
+Aggregate
+  ->  Seq Scan on date_smalldatetime_cmp_t1
+        Filter: (date_col >= '2023-06-16 00:00:00'::smalldatetime(0) without time zone)
+~~END~~
+
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.246 ms
+~~END~~
+
+
+SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col BETWEEN CAST('2023-06-16' AS SMALLDATETIME) AND CAST('2023-06-18' AS SMALLDATETIME);
+GO
+~~START~~
+text
+Query Text: SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE date_col BETWEEN CAST('2023-06-16' AS SMALLDATETIME) AND CAST('2023-06-18' AS SMALLDATETIME)
+Aggregate
+  ->  Seq Scan on date_smalldatetime_cmp_t1
+        Filter: ((date_col >= '2023-06-16 00:00:00'::smalldatetime(0) without time zone) AND (date_col <= '2023-06-18 00:00:00'::smalldatetime(0) without time zone))
+~~END~~
+
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.201 ms
+~~END~~
+
+
+SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE CAST('2023-06-14' AS SMALLDATETIME) > date_col;
+GO
+~~START~~
+text
+Query Text: SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE CAST('2023-06-14' AS SMALLDATETIME) > date_col
+Aggregate
+  ->  Seq Scan on date_smalldatetime_cmp_t1
+        Filter: ('2023-06-14 00:00:00'::smalldatetime(0) without time zone > date_col)
+~~END~~
+
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.176 ms
+~~END~~
+
+
+SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE CAST('2023-06-14' AS SMALLDATETIME) >= date_col;
+GO
+~~START~~
+text
+Query Text: SELECT COUNT(*) FROM date_smalldatetime_cmp_t1 WHERE CAST('2023-06-14' AS SMALLDATETIME) >= date_col
+Aggregate
+  ->  Seq Scan on date_smalldatetime_cmp_t1
+        Filter: ('2023-06-14 00:00:00'::smalldatetime(0) without time zone >= date_col)
```

Comment:
we should have tests where index scan is being picked

Reply:
This is the the testing where we create index on date and then compare smalldatetime and date. 
Already filed [jira](https://jira.rds.a2z.com/browse/BABEL-5428) for this.